### ISSUE TITLE
Fresh install getting Error: Module version mismatch. Expected 48, got 46.

### ISSUE THREAD

**Issue by brendanstennett** [GoogleCloudPlatform/cloud-functions-emulator#132]
### Description

Just installed functions-emulator and running any command gives the error:
```
Error: Module version mismatch. Expected 48, got 46.
    at Error (native)
    at Object.Module._extensions..node (module.js:597:18)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/@google-cloud/functions-emulator/node_modules/grpc/src/node/src/grpc_extension.js:45:15)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
```

### Steps to reproduce

0. Nodejs installed and running v6.11.2
1. `npm install -g @google-cloud/functions-emulator`
2. `functions-emulator start` gives the above error

Thanks!

**Comment by jmdobry** [comment]
Give you give the fix I describe in [#126](https://github.com/GoogleCloudPlatform/cloud-functions-emulator/issues/126#issuecomment-319708141) a try?

**Comment by brendanstennett** [comment]
Yes I got this working based on that.  Had to uninstall then reinstall.

Thanks!
